mungegithub/mungers/testowner/owner.go

```diff
@@ -41,18 +42,28 @@ func normalize(name string) string {
 // OwnerList uses a map to get owners for a given test name.
 type OwnerList struct {
 	mapping map[string]string
+	rng     *rand.Rand
 }
 
-// TestOwner returns the owner for a test, or the empty string if none is found.
+// TestOwner returns the owner for a test, an owner from $default if present,
+// or else the empty string if none is found.
 func (o *OwnerList) TestOwner(testName string) string {
 	name := normalize(testName)
 	owner, _ := o.mapping[name]
+	if owner == "" {
+		owner, _ = o.mapping["$default"]
+	}
+	if strings.Contains(owner, "/") {
```

Comment:
It's in a CSV, so commas are right out. The / is more visible/understandable than a tab.